SEED-platform/seed
# Add Terms of Service

#### What's this PR do?
Add Legal Terms to landing page
#### How should this be manually tested?
go to the login page or the "home" page when not logged in.  you should see:

![Screen Shot 2023-05-30 at 2 45 07 PM](https://github.com/SEED-platform/seed/assets/2205659/c74c37c6-9aba-4535-b74f-cf94caa07975)

Then click on "View/Hide Terms" to reveal:
![Screen Shot 2023-05-30 at 2 45 18 PM](https://github.com/SEED-platform/seed/assets/2205659/92b72771-d481-4b4f-870c-dcd539ef7f20)

#### What are the relevant tickets?
#3964
